docker/apply/forms-json/lawyers.json

```diff
@@ -28,7 +28,7 @@
       "controller": "./pages/start.js"
     },
     {
-      "title": "Check your answers",
+      "title": "Update your information",
```

Comment:
new users will end up seeing "update your information" which will be wrong. 

see `getNewSessionWebhookData`

You can customise/overwrite certain strings in the customText object.

Reply:
Ah thanks for spotting this, it was left in by accident. Only the .json file that are prefixed with annual-review should have that title 😅